### PR TITLE
prototype of cardless voting with poll worker activation

### PR DESCRIPTION
**Pull request by benadida**

### CODE REVIEW

**Comment by benadida** [comment]
@beausmith this is just FYI, not trying to merge this as is, of course.

File: apps/bmd/src/AppRoot.tsx [hunk 1]
```diff
@@ -809,6 +831,32 @@ class AppRoot extends React.Component<Props, State> {
...
 
+  public pollWorkerActivateBallot = (ballotStyleId: string) => {
+    this.setState((prevState) => {
+      const election = prevState.electionDefinition!.election
+      const ballotStyle = getBallotStyle({
+        election: prevState.electionDefinition!.election,
+        ballotStyleId,
+      })
+
+      if (election && ballotStyle) {
```
Comment (beausmith) [inline apps/bmd/src/AppRoot.tsx:842]: `election` should always exist at this point no?

File: apps/bmd/src/pages/PollWorkerScreen.tsx [hunk 2]
```diff
@@ -95,12 +105,61 @@ const PollWorkerScreen = ({
...
+
+  if (cardlessActivatedBallotStyleId) {
+    return (
+      <React.Fragment>
+        <Screen>
+          <Main>
+            <MainChild centerVertical maxWidth={false}>
+              <Prose textCenter>
+                <h1 aria-label="Ballot Style Activated">
+                  Ballot Style {cardlessActivatedBallotStyleId} Activated
```
Comment (beausmith) [inline apps/bmd/src/pages/PollWorkerScreen.tsx:122]:
```suggestion
                  Ballot Style “{cardlessActivatedBallotStyleId}” Activated
```

File: apps/bmd/src/pages/PollWorkerScreen.tsx [hunk 3]
```diff
@@ -95,12 +105,61 @@ const PollWorkerScreen = ({
...
+                </h1>
+                <p>Remove poll worker card and let the voter vote.</p>
```
Comment (beausmith) [inline apps/bmd/src/pages/PollWorkerScreen.tsx:124]:
```suggestion
                <p>Remove the poll worker card and instruct the voter to press the “Start Voting” button.</p>
```

**Comment by beausmith** [comment]
Closing in exchange for #148

**Comment by beausmith** [comment]
Closing in exchange for #148

**Pull request closed without merging**